Federal Data/Nepal Rastra Bank/Active saving accounts of 7 province.csv

```diff
@@ -0,0 +1,9 @@
+State/Province,Total number of active male accounts,Total number of active female accounts,Total population,Percentage of active male accounts,Percentage of active female accounts
+(All),9168740,5816647,22183283,259,169

```

Comment:
I believe the percentages for the first column are incorrect. They are both over 100%. It looks like they were derived from adding the percentages in the provinces together rather than calculating percentages for the full population.

What do "Percentage of active male accounts" and "Percentage of active male accounts" mean in these data? Is it:
percent of accounts that are held by men and by women and that are "active"
percent of men (or women) in the total population who own a savings account?

What does "active" mean in these data? That isn't something that would be useful to answer when the data are used.